CONTRIBUTING.md

```diff
@@ -177,7 +176,7 @@ If you want to user another DockerHub repository you need to set `DOCKER_HUB_REP
   - `GOVC_URL` same as `ESX IP`
   - `GOVC_USERNAME` & `GOVC_PASSWORD`: user credentials logging in to `ESX IP`
 
-- You **need** to set following environment variables to run swarm cluster related tests.
+- You **need** to set following environment variables to run swarm cluster related tests. You **need** to configure swarm cluster in order to run swarm related testcase otherwise there will be a test failure.
```

Comment:
I have added a `TODO` information ... and will work on simplifying the `make test-all` invocation. It will be handled separately in different PR having it said keeping this PR only for documentation changes.